src/evt/si.c

```diff
@@ -1175,7 +1178,336 @@ s32 func_802C73B8(ScriptInstance* script) {
     return 1;
 }
 
-INCLUDE_ASM(s32, "evt/si", si_execute_next_command);
+s32 si_execute_next_command(ScriptInstance *script) {
+    while (TRUE) {
+        s32 status = ApiStatus_DONE2;
+        s32* lines;
+        s32 argC;
```

Comment:
Call this nargs to be consistent with funcs lower down in the file? Or change nargs to argc